### DISCUSSION TITLE
Disabling the event end time

### DISCUSSION THREAD

**Discussion by zbyszekgdynia** [totaldebug/atomic-calendar-revive#1528] [Feature Requests]
I couldn't find a function to disable the event end time. This is important when triggering via an automation calendar. Such an action is supposed to occur once at a specified time, for example, at 5:30. In Google Calendar, I can set it to last 0 minutes, but in Home Assistant, it will be half an hour.

**Comment by marksie1988** [comment]
This card has nothing to do with automations, this will be the HA Calendar integration that you need to ask about.

**Comment by marksie1988** [comment]
`showEndTime: false` already does this — see [main configuration docs](https://github.com/totaldebug/atomic-calendar-revive/blob/beta/docs/configuration/main.rst). When set, only the start time is shown for each event. Closing as already implemented.
